Tera/View/SoundWaveView.swift

```diff
@@ -17,7 +17,12 @@ struct SoundWaveView: View {
     
     // 2
     private func normalizeSoundLevel(level: Float) -> CGFloat {
-        let level = (max(30, CGFloat(level) + 50) / 2) - 14.9 
+//        let level = (max(14, CGFloat(level) + 50) / 2) - 6.9
+        if level == 0 {
```

Comment:
no need to change, just adding my opinion

untuk logic yang edge case gini, idealnya biasanya dikasih comment gitu supaya other engineer bisa tau konteksnya misal
```swift
/**
  guard the normalization to prevent auto scalling 
  at the first time active the microphone monitor
 */
if level == 0 {
  return CGFloat(0.1)
}
```

kadang note2 seperti ini berguna buat dikemudian hari (biar kita ga lupa kalo kita balik lagi ke code ini)

Reply:
Done. Thanks for the review